WSC.js

```diff
@@ -20,16 +20,10 @@ WSC.DirectoryEntryHandler = handlers.DirectoryEntryHandler
 WSC.BaseHandler = handlers.BaseHandler
 
 var main_fs = new WSC.FileSystem(__dirname)
-main_fs.getByPath('/directory-listing-template.html', function(file) {
-    file.text(function(data) {
-        WSC.template_data = data
-    })
-})
-main_fs.getByPath('/directory-listing-template-static.html', function(file) {
-    file.text(function(data) {
-        WSC.static_template_data = data
-    })
-})
+const fs = require('fs')
+const path = require('path')
```

Comment:
the path and fs libraries are defined on global (see index.js)

Reply:
Good catch, thanks!